app/views/comments/_edit.html.erb

```diff
@@ -1,15 +1,50 @@
-<form id="c<%= comment.id %>edit" style="display:none;" class="well" action="/comment/update/<%= comment.id %>">
```

Comment:
Or, perhaps, since the distinction is not whether it's a question, but if it's formattable, we could have it switch based on a local variable like `richText` which is part of the `render partial` call from a higher template. 

Is the rich formatting toolbar used in other places where we could reduce duplication? Not essential, but if possible it'd be great to reduce duplication -- especially as the new rich editor may need to cleanly replace some of this someday down the road.